Mark example as obsolete (#5)
DevExpress-Examples/obsolete-blazor-DxDateEdit-date-range-picker: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T809157)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
[![](https://img.shields.io/badge/💬_Leave_Feedback-feecdd?style=flat-square)](#does-this-example-address-your-development-requirementsobjectives)
<!-- default badges end -->

# Blazor Date Edit - Implement the date range selector

⛔ DEPRECATED. This example was deprecated. 

> In v24.1.3 and higher, we implemented the [Date Range Picker](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateRangePicker-1) editor that allows users to select date ranges. Please check the [Date Range Picker demo](https://demos.devexpress.com/blazor/DateRangePicker) for details.

Reviewer: ```suggestion
Beginning with v24.1.3 and onward, we implemented the [Date Range Picker](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateRangePicker-1) editor that allows users to select date ranges. Please check the [Date Range Picker demo](https://demos.devexpress.com/blazor/DateRangePicker) for details.

```
